src/index.ts

```diff
@@ -1,3 +1,5 @@
+import './tokens-dist/css/variables.css';
```

Comment:
this brings in the tokens that are css custom properties into the built stylesheet